src/components/Page.js

```diff
@@ -1,16 +1,37 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { Box } from 'grommet';
 import Nav from './Nav';
 
-const Page = ({ children, background }) => (
-  <Box pad="large" background={background}>
-    <Box>
-      <Nav />
-      <Box margin={{ top: 'large' }}>{children}</Box>
-    </Box>
-  </Box>
-);
+export default class Page extends Component {
+  componentDidMount() {
+    this.scrollToSection();
+  }
 
-Page.propTypes = Box.propTypes;
+  componentDidUpdate() {
+    this.scrollToSection();
+  }
+
+  scrollToSection = () => {
+    const name = window.location.hash.split('#')[1];
+    if (name) {
+      document.getElementById(name).scrollIntoView({
```

Comment:
Doesn't the browser already do this?
https://stackoverflow.com/questions/19299220/is-there-an-event-that-fires-when-a-browser-scrolls-to-a-named-anchor

Reply:
The browser supplied event doesn't fire when you go to a url with the hash already set. It only works if you are already on the same page